Makefile

```diff
@@ -59,6 +59,8 @@ test: deps
 
 # Include thematic Makefiles
 include Makefile-*.mk
```

Comment:
This line, together with `Makefile-package.mk` and `Makefile-docker.mk` should be removed.

Reply:
you are right! I'll garbage collect everything!